.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/NCAR/ccpp-framework
 [submodule "ccpp-physics"]
 	path = ccpp/physics
-	url = https://github.com/NCAR/ccpp-physics
+	url = https://github.com/pjpegion/ccpp-physics
```

Comment:
This change needs to be reverted. It should still point to the NCAR fork.

Reply:
Done